h2o-r/h2o-package/R/models.R

```diff
@@ -2498,6 +2498,71 @@ h2o.get_regression_influence_diagnostics <- function(model, predictorSize = -1)
     }
 }
 
+#'
+#' Extracts the final training negative log likelihood of a GLM model.
+#'
+#' @param model an \linkS4class{H2OModel} object.
+#' 
+#' @examples 
+#' \dontrun{
+#' library(h2o)
+#' h2o.init()
+#' 
+#' f <- "https://s3.amazonaws.com/h2o-public-test-data/smalldata/junit/cars_20mpg.csv"
+#' cars <- h2o.importFile(f)
+#' predictors <- c("displacement", "power", "weight", "acceleration", "year")
+#' response <- "acceleration"
+#' cars_model <- h2o.glm((y=response, 
+#'                        x=predictors, 
+#'                        training_frame = cars, 
+#'                        family="gaussian",
+#'                        generate_scoring_history=TRUE)
+#' objValue <- h2o.negative_log_likelihood(cars_model)
+#' }
+#' @export 
+h2o.negative_log_likelihood <- function(model) {
+    return(h2o.extract_value(model, "negative_log_likelihood"))
+}
+
+#'
+#' Extracts the final training average objective function of a GLM model.
+#'
+#' @param model an \linkS4class{H2OModel} object.
+#' 
```

Comment:
CRAN might be require return value specification as well.

Reply:
Added it.  Thanks for the catch